setup.py

```diff
@@ -21,6 +21,21 @@
 from setuptools import setup, Extension
 import platform
 import os
+import subprocess
+import re
+
+def get_mac_include_dirs():
+    cmd = "brew list --versions hunspell | tr ' ' '\n' | tail -1"
+    ps = subprocess.Popen(cmd,shell=True,stdout=subprocess.PIPE,stderr=subprocess.STDOUT)
```

Comment:
I should setup some CI on this but could you pep8-ify your PR? 
Space after comma here.

I don't see why you need `shell=True`.